.github/actions/test-linux/action.yml

```diff
@@ -18,10 +18,15 @@ runs:
         export PDO_DBLIB_TEST_USER="pdo_test"
         export PDO_DBLIB_TEST_PASS="password"
         export SKIP_IO_CAPTURE_TESTS=1
+        export TEST_PHP_JUNIT=junit.out.xml
         sapi/cli/php run-tests.php -P -q ${{ inputs.runTestsParameters }} \
           -j$(/usr/bin/nproc) \
-          -g FAIL,XFAIL,BORK,WARN,LEAK,XLEAK,SKIP \
+          -g FAIL,BORK,WARN,LEAK,XLEAK \
           --offline \
           --show-diff \
           --show-slow 1000 \
           --set-timeout 120
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        path: ${{ github.workspace }}/junit.out.xml
```

Comment:
You might need `name:` to disambiguate the different jobs, otherwise the artifacts appear to overwrite each other? At least I'm seeing only one, but two successful Linux jobs.

Reply:
Yeah, I was hoping it would do it automatically but apparently not...

Reply:
Yeah, either name the packages differently (like done in the meantime), or name junit.out.xml differently (in which case you get a single archive with multiple test results). I don't have a preference.

Reply:
Yeah I opted for multiple archives but it's probably not super important.